crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -1027,9 +1046,10 @@ impl CheckpointBuilder {
         let new_checkpoint = self
```

Comment:
optional nit- while you're in the neighborhood, maybe fix this name to `new_checkpoints`?